tfjs-backend-wasm/starter/parcel/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@tensorflow/tfjs-backend-wasm": "3.14.0",
-    "@tensorflow/tfjs-core": "3.14.0"
+    "@tensorflow/tfjs-core": "4.5.0"
```

Comment:
why do we still update this version? I thought it's fixed in the previous version updater PR.

Reply:
Oh, #7550 hasn't been merged yet. Let me try again on that branch.